hedera-mirror-rest/monitoring/monitor_apis/account_tests.js

```diff
@@ -187,6 +187,72 @@ const getSingleAccount = async (server) => {
   };
 };
 
+/**
+ * Verify that we return token relationships for an existing account
+ * @param {Object} server API host endpoint
+ */
+const getSingleAccountTokenRelationships = async (server) => {
+  // Call the get /tokens endpoint to get a list of tokens.
+  const tokensPath = '/tokens';
+  const tokensJsonRespKey = 'tokens';
+  const tokenMandatoryParams = ['token_id', 'symbol', 'admin_key'];
```

Comment:
We don't need the other fields for this test.
```suggestion
  const tokenMandatoryParams = ['token_id'];
```